frog-utils/src/types.js

```diff
@@ -135,7 +135,8 @@ export type ActivityPackageT = {
     initData: any
   },
   exportData?: (config: Object, product: activityDataT) => string,
-  formatProduct?: (config: Object, item: any) => any
+  formatProduct?: (config: Object, item: any) => any,
+  ConfigComponent?: ReactComponent<any>
```

Comment:
It makes sense to me that this component has the props `configData` and `setConfigData`. Might be useful to add it to the type definition